.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Pull Request Template
+1. Copy the template from below
+2. Replace this text with the template
+3. Fill in template
+4. Select "Create pull request"
+    
+## :memo: :lipstick: :new: :bomb: :bug: :art: Template :memo: :lipstick: :new: :bomb: :bug: :art: 
+* Fill out the template below
+* [Link to any relevant issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue)
+
+**Delete this and everything above**
```

Comment:
Not really a fan of essentially making the templates free form -- "delete everything" means no template. Imo, better is to make an actual template with little to remove/fill in. The PR template I use at work is more like:

### Background
**_A sentence or two about why this change is needed and how we got here. More than an issue link, though that is sufficient if the issue has a lot of context._**

### Issues
* links to issues or related PRs
* (though, this can be in the background, at work we don't have the same kind of issue linking as github, so we add explicit hyperlinks)

### Notes
**_Implementation specific notes, such as design decisions made or techical limitations you faced._**

### Testing Done
```sh
./gradlew clean build test site
```
**_Include any manual testing you've done._**



Naturally, it doesn't quite fit the vibe you have, like the emojis. Changes will be needed. Maybe some stuff about the title too.